tasks/bareos_server.yml

```diff
@@ -163,15 +155,6 @@
     mode: '0640'
   loop: "{{ bareos_jobs }}"
 
-- name: Create bareos client config files
-  template:
-    src: bareos-dir/client/client.conf.j2
-    dest: /etc/bareos/bareos-dir.d/client/{{ item.name }}.conf
```

Comment:
Will there be some leftover files with this change?

It looks like the client will create a file called `myself.conf` while the task remove here was creating `{{ item.name }}.conf` for each clients, on the server. Is that correct?
https://github.com/mila-iqia/ansible-role-bareos/blob/dfaeecba650c953417f3caa7ad09d04c6f5a2652/tasks/register_client.yml#L45

Reply:
My bad, the duplication might be with this task.
https://github.com/mila-iqia/ansible-role-bareos/blob/dfaeecba650c953417f3caa7ad09d04c6f5a2652/tasks/register_client.yml#L2-L13 which creates `/etc/bareos/bareos-dir.d/client/{{ item.name }}.conf` on the server using `bconsole` instead of a template.

Is that correct?

Reply:
No, there won't be any leftover : `{{ item.name }}.conf ` are still created, by another task :

https://github.com/mila-iqia/ansible-role-bareos/blob/mila/tasks/register_client.yml#L11